dappnode_package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hopr.public.dappnode.eth",
-  "version": "1.0.27",
+  "version": "1.0.19",
```

Comment:
Why changing the version to a lower value ?

Reply:
This is the actual value that is published into the Dappstore.
So no matter how much we increment it here, the value that makes it into the package will be always fetched from the Dappstore and incremented. 

So I set the value to be in agreement with what we have published as of now.